h_malloc.c

```diff
@@ -903,7 +903,10 @@ static int regions_grow(void) {
         }
     }
 
-    memory_map_fixed(ra->regions, ra->total * sizeof(struct region_metadata));
```

Comment:
This call shouldn't be checked because failure doesn't impact whether the function succeeded. It doesn't really matter if it succeeds. The point of doing this is to free memory.